src/ape_accounts/config.py

```diff
@@ -0,0 +1,21 @@
+from enum import Enum
+
+from ape.api.config import PluginConfig
+
+
+class CalldataRepr(str, Enum):
+    full = "full"
+    abridged = "abridged"
```

Comment:
I plan to add "decoded" here and it will attempt to decode the calldata if possible

Reply:
decoded would be amazing tbh

@BowTiedDevil was saying the same yesterday

Reply:
It'll be a follow up to this!